test_scripts/RC/Capabilities/018_ControlCapabilities_Empty_array_RADIO.lua

```diff
@@ -0,0 +1,51 @@
+---------------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SDL_RC] Capabilities
+--
+-- Description:
+-- In case:
+-- HMI respond with empty array for module to RC.GetCapabilities request from SDL
+--
+-- SDL must:
+-- Reject all RPCs from mobile application to such module with result code UNSUPPORTED_RESOURCE, success:false
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonRC = require('test_scripts/RC/commonRC')
+
+--[[ Local Variables ]]
+local disabledModule = "RADIO"
+local enabledModule = "CLIMATE"
```

Comment:
@dboltovskyi ,  think better to avoid this variables disabledModule and enabledModule.
It decreasy readability. It maek sense if in one test you will use combinations but here strict case - CLIMATE, enabled, RADIO is disabled. Then in getHMIParams just use radioControlCapabilities and in scenario just readable strings.
the same for script 019